App/screens/photos.tsx

```diff
@@ -113,7 +113,8 @@ class Photos extends Component<Props> {
         />
       )
     } else {
-      const { photo, error } = row.item.processingPhoto
+      const { photo, state, error } = row.item.processingPhoto
+      const color = state === 'pending' ? 'white' : 'blue'
```

Comment:
Display a white circle on `pending` photos, blue circle on photos that are currently being processed.